index.html

```diff
@@ -1,199 +1,172 @@
+                        personalized reads; no accounts or 
+                        sign-ups required.</p>
+                    </section>
+                    <section class="part-2">
+                        <section class="but">
+                            <section class="buthtml"><a class="htmlbut">HTML</a></section>
+                            <section class="butcss"><a class="cssbut">CSS</a></section>
+                            <section class="butjs"><a class="jsbut">JavaScript</a></section>
+                        </section></section>
+                    <button class="see">See project</button>
+                </section>
+                </section>
+                <section class="card">
+                    <section class="part-2">
+                    <section class="part-2">
+                        <img class="Portfolio" src="Portfolio2.png" alt="Feyisa">
+                    </section>
+                    <section class="part-2">
+                        <h1 class="tonic">Multi-Post
+                            Stories</h1>
+                    </section>
+                    <section class="part-2" >
+                    <ul class="can">
+                        <li class="can"><a class="canopy">CANOPY</a></li>
+                        <li class="can"><img class="counter" src="point.png"></li>
+                        <li class="can" ><a class="back" >Back End Dev</a></li>
+                        <li class="can"><img class="counter" src="point.png"></li>
+                        <li class="can" ><a class="num">215</a></li>
+                    </ul>
+                    </section>
+                    <section class="part-2"><p class="daily">A daily selection of privately 
+                        personalized reads; no accounts or 
+                        sign-ups required.</p>
+                    </section>
+                    <section class="part-2">
+                        <section class="but">
+                            <section class="buthtml"><a class="htmlbut">HTML</a></section>
+                            <section class="butcss"><a class="cssbut">CSS</a></section>
+                            <section class="butjs"><a class="jsbut" >JavaScript</a></section>
+                        </section></section>
+                    <button class="see">See project</button>
+                </section>
+                </section>
+                <section class="about-page">
+                <section class="about">
+                    <h2 class="myself">About Myself</h2>
+                    <section class="about">
+                        <p class="self">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p></section>
+                    <section class="let">LET’S CONNECT</section>

```

Comment:
- [ ] I kindly suggest that you use a header e.g h1 - h6 tag for this line instead of a section tag